docs/Using_Datasets.md

```diff
@@ -118,7 +117,25 @@ dataset is linked to a publication listed in PubMed.*
 -------------
 
 
-Step 3: Data Scopes
+
+Step 3: Using Dataset favorites
+---------------
+
+Since R2 is hosting hundreds of datasets it could be handy to store in a quick way the datasets you often use. Clicking on the starred+ symbol will add the dataset you currently selected to a subset of favorite datasets.
+
+![](_static/images/UsingDatasets_favorites.png "Figure 7: Adding datasets to favorites")
+	
+[**Figure 6: Adding datasets to favorites**](_static/images/UsingDatasets_favorites.png)
+
+	
+You can delete datasets from you favorite lists by clicking the delete symbol.  Clicking the "select from favorites" link gives a pop-up where you can select a dataset from your list of favorites.
```

Comment:
you can remove datasets from your favourites list by simply clicking on the remove symbol behind a favourite dataset